UltimateWoltlabMaster.sh

```diff
@@ -0,0 +1,123 @@
+reset
+
+echo "UltimateWoltlabMaster von Justman10000"
+echo "Dies ist unter der GNU General Public License v3.0 lizenziert"
+
+echo "Was soll getan werden?"
+echo "Installieren [1]"
+echo "Deinstallieren [2]"
+echo "Zurück [3]"
```

Comment:
Also, you could try keeping your script a little smaller by using `echo -e "text"` instead of creating a new `echo` for each new line.